### ISSUE TITLE
SDK folder

### ISSUE THREAD

**Issue by saha-anik** [VitorVidal/OpenBR4Net#1]
I try to build openbr. I am confused by sdk folder. Do you mean this folder==>
**ParentFolder\openbr\build**  or **ParentFolder\openbr\build\openbr**

**Comment by VitorVidal** [comment]
Hello saha-anik, 
Looking at the code, sounds that I forgot to upload the "Install\SDK" folder at on project OpenBR4Net vc++. I don't have it anymore (it was four years ago) but you can follow the instructions on "http://openbiometrics.org/docs/install/#windows" build a openbr for windows.

**Comment by saha-anik** [comment]
Yes I have build openbr. Run it from command promt. But the fact is 
In header file you have declared **msvc2012/openbr/openbr_plugin.h** in OpenBrNet.h and there is  another error ==>**The command "xcopy "D:\Office\OpenBR4Net\OpenBR4Net\Install\SDK" "D:\Office\OpenBR4Net\OpenBR4Net.App.Console\x64\bin\Debug\OpenBR4Net" /S /Y /I
xcopy "D:\Office\OpenBR4Net\x64\Debug\OpenBR4Net.dll" "D:\Office\OpenBR4Net\OpenBR4Net.App.Console\x64\bin\Debug\OpenBR4Net" /S /Y /I**

That actually confused me how to put this project structure in openbr.

**Comment by VitorVidal** [comment]
For the first error you need to declare the path to "openbr_plugin.h" on openbr source, "msvc2012" was my path to openbr source. So you should change that line to reflect you openbr source folder.
For the second error: it's comes from a pré-build event on "OpenBR4Net.App.Console" project, it should copy all the files from "Install\SDK" and "OpenBR4Net.dll" files to it's debug folder before build. You can remove it and copy the files manually.
